Info/index.ts

```diff
@@ -0,0 +1,22 @@
+import { AzureFunction, Context } from "@azure/functions";
+import * as express from "express";
+import { secureExpressApp } from "io-functions-commons/dist/src/utils/express";
+import { setAppContext } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import createAzureFunctionHandler from "io-functions-express/dist/src/createAzureFunctionsHandler";
+import { Info } from "./handler";
+
+// Setup Express
+const app = express();
+secureExpressApp(app);
+
+// Add express route
+app.get("/api/v1/info", Info());

```

Comment:
should be `/info` afaik, see comment above: 

https://github.com/pagopa/io-functions-admin/blob/master/Info/index.ts#L13

Reply:
not relevant, see comments above